mimic/canned_responses/mimic_presets.py

```diff
@@ -19,5 +19,19 @@
                            "invalid_flavor_ref": ["INVALID-FLAVOR-ID", "8888", "-4", "1"],
                            "server_error": "sets server state to error on create",
                            "server_building": "sets the server to be in building state for given time"
-                                              " in seconds"}
-               }
+                                              " in seconds"},
+               "identity": {
+                   "maas_admin_roles": [
+                       "this_is_an_impersonator_token",
+                       "this_is_an_impersonator_token_also",
+                       "impersonate_watson",
+                       "impersonate_creator",
+                       "this_is_an_impersonator_token_also_2",
+                       "impersonate_foo_token"],
+               "racker_token": ["this_is_a_racker_token"],
```

Comment:
Non-blocking - I think the indentation starting from here might be kind of messed up (should it be lined up with `"maas_admin_roles"`?)